test/integration/repos_test.go

```diff
@@ -135,19 +131,17 @@ func TestRepositories_EditBranches(t *testing.T) {
 		t.Fatalf("Repositories.UpdateBranchProtection() returned error: %v", err)
 	}
 
+	enforcementURL := "https://api.github.com/repos/" + *repo.Owner.Login + "/" + *repo.Name + "/branches/master/protection/enforce_admins"
 	want := &github.Protection{
 		RequiredStatusChecks: &github.RequiredStatusChecks{
 			Strict:   true,
 			Contexts: []string{"continuous-integration"},
 		},
 		RequiredPullRequestReviews: &github.PullRequestReviewsEnforcement{
-			DismissalRestrictions: github.DismissalRestrictions{
-				Users: []*github.User{},
-				Teams: []*github.Team{},
-			},
 			DismissStaleReviews: true,
 		},
 		EnforceAdmins: &github.AdminEnforcement{
+			URL:     &enforcementURL,
```

Comment:
You can use `String` method to create a `*string` inline here. I think it'd be simpler/more readable that way.